src/python/org_fetcher/fetch_orgs.py

```diff
@@ -0,0 +1,64 @@
+from collections import defaultdict
+import csv
+import os
+
+PATH_TO_DATAFILES = os.path.join(
+    '..', '..', 'php', 'gov-harvester', 'datafiles'
+)
+
+
+orgs = defaultdict(lambda: 0)
+for filename in os.listdir(PATH_TO_DATAFILES):
+    filename_lower = filename.lower()
+    if not filename_lower.endswith('.csv'):
```

Comment:
I think we could extract a method `is_csv(filename)` to make the intent a bit more obvious and potentially make the following comment obsolete.